docs/dg/report.md

```diff
@@ -20,7 +20,7 @@ The following is a snapshot of the report:
 
 ## Report architecture
 
-![report architecture](../images/report-architecture.png)
+<puml src="../diagrams/ReportArchitecture.puml"/>
```

Comment:
![image](https://user-images.githubusercontent.com/39231750/104844507-97ba1880-590b-11eb-9b6b-3099c7751b1e.png)

Reply:
Thanks @0blivious for the review!

I believe this error is caused due to Graphviz not being installed.
May I know the steps to replicate this?

It worked well when I staged the site locally using `markbind serve`:
![image](https://user-images.githubusercontent.com/59356158/104847399-b70c7200-591a-11eb-952b-18568dbad14b.png)

I believe this issue might be related to https://github.com/MarkBind/markbind/issues/916.

Reply:
You can always check the deployment at the check section https://docs-1406-pr-reposense-reposense.surge.sh/. Your ReportArchitectureAuthorship.puml is still working. Please help investigate why only the first puml is not displayed?

Reply:
With reference to https://stackoverflow.com/questions/54413962/cant-render-component-diagram-in-plantuml-intllij, it seems Graphviz installation on the deployment server is required for rendering of PlantUML **component diagrams**. **Sequence and activity diagrams** (`ReportArchitectureAuthorship.puml` and `ReportArchitectureSummary.puml`) work without Graphviz installation. 

I believe fixing #1427 will solve the displaying of `ReportArchitecture.puml` on the MarkBind generated website.

Reply:
In that case, we might want to replace the puml diagram back to pug file as we might not want to increase the build time. Also, you may wish to file a bug/feature request to the MarkBind repo, asking them to investigate/address the issue.

@fzdy1914 how do you feel about the Graphviz installation?

Reply:
If we already start using Markbind, using Graphviz is kind of acceptable to me.

Reply:
If only one diagram is affected, you can also generate the diagram locally and add that to the repo (while keeping the puml code as well), to reduce the build time. Seems wasteful to install Graphwiz for every PR build when it is used in only one diagram which is unlikely to be modified in most PRs.